test/integration.js

```diff
@@ -205,6 +220,57 @@ tape('health works correctly', function(t) {
 	.catch(err => t.fail(err))
 })
 
+tape('heartbeat works correctly', function(t){
+	Promise.resolve()
+	.then(() => wait(waitTime)) // wait till a new state is schedule to be produced
+	.then(function() {
+		return fetch(`${followerUrl}/channel/${dummyVals.channel.id}/validator-messages/${dummyVals.ids.follower}/HeartBeat?limit=1`)
```

Comment:
this is very repetitive, can you refactor it; see the tests for submitting invalid events and etc.

Reply:
yeah definitely